packages/ERTP/src/amountMath.js

```diff
@@ -77,9 +80,14 @@ const helpers = {
 };
 
 /**
- * @template {AmountValue} V
- * @type {(value: V) => AssetKindForValue<V>}
+ * @type {{
+ *   (value: CopyBag): 'copyBag';
+ *   (value: CopySet): 'copySet';
+ *   (value: Array): 'copySet';
+ *   (value: bigint): 'nat';
+ * }}
  */
+// @ts-expect-error cast
```

Comment:
What are we casting to what?

For each `@ts-expect-error`, I'm trying to understand why we're certain that the declared type is correct.